File: scratch/find_cms_keys.js
const fs = require('fs');

const content = fs.readFileSync('C:\\Users\\edarj\\.gemini\\antigravity\\scratch\\grade-management-system\\src\\components\\WebsiteCMS.jsx', 'utf8');
const lines = content.split('\n');

lines.forEach((line, index) => {
  if (line.includes('site_settings') || line.includes('our_divisions') || line.includes('hero_styling')) {
    console.log(`Line ${index + 1}: ${line.trim()}`);
  }
});

File: scratch/find_cms_keys.py
import re

with open(r'C:\Users\edarj\.gemini\antigravity\scratch\grade-management-system\src\components\WebsiteCMS.jsx', 'r', encoding='utf-8') as f:
    lines = f.readlines()

for i, line in enumerate(lines):
    if 'site_settings' in line or 'our_divisions' in line or 'hero_styling' in line:
        print(f"Line {i+1}: {line.strip()}")
